vertica_python/vertica/messages/frontend_messages/startup.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, user, database, options=None):
         self._version = vertica_python.__version__.encode(ASCII)
         self._platform = platform.platform().encode(ASCII)
         self._pid = '{0}'.format(os.getpid()).encode(ASCII)
-        self._label = self._type + b'-' + self._version + b'-' + str(uuid.uuid1()).encode(ASCII)
+        self._label = self._type + b'-' + self._version + b'-' + str(uuid.uuid1()).encode(ASCII) if label is None else label
 
```

Comment:
Please rebase your branch, see instructions in `CONTRIBUTING.md`: https://github.com/vertica/vertica-python/blob/master/CONTRIBUTING.md#step-5-push-and-rebase